modules/authentication/src/handlers/local.ts

```diff
@@ -216,6 +217,86 @@ export class LocalHandlers {
     };
   }
 
+  async authenticateWithPhone(call: ParsedRouterRequest): Promise<UnparsedRouterResponse> {
+    if (!this.initialized)
+      throw new GrpcError(status.NOT_FOUND, 'Requested resource not found');
+    const { phone } = call.request.params;
+    const context = call.request.context;
+    const clientId = context.clientId;
+    const config = ConfigController.getInstance().config;
+    if (isNil(context))
+      throw new GrpcError(status.UNAUTHENTICATED, 'No headers provided');
+    let user: User | null = await User.getInstance().findOne(
+      { phoneNumber: phone },
+    );
+    if (isNil(user)) {
+      user = await User.getInstance().create({
+        phoneNumber: phone,
+        isVerified: false,
```

Comment:
verified should be true. As with the email login should not be permitted for a user if their credentials are not verified.